api/src/main/java/ai/djl/inference/Predictor.java

```diff
@@ -95,6 +95,7 @@
     public Predictor(Model model, Translator<I, O> translator, boolean copy) {
         this.model = model;
         this.manager = model.getNDManager().newSubManager();
+        this.manager.setName("Predictor");
```

Comment:
maybe use lowercase to align with others
```suggestion
        this.manager.setName("predictor");
```